release/cloudbuild_presubmit.yaml

```diff
@@ -0,0 +1,91 @@
+steps:
+  # First create a fake tag we'll use throughout the CI build process below.
+  # Unfortunately, GCB has no concept of dynamically creating substitutions or
+  # passing ENV vars between steps, so the best we can do is to create a file
+  # containing our tag in the share workspace which other steps can inspect.
+  - name: bash
+    script: |
+      date +'0.2.%s-incompatible' > /workspace/fake_tag
```

Comment:
Why is this `0.2.xxx` ? That 2 feels like a magic number that will pop up as a red herring in the future. Suggest `0.0.xxx` if that's a possibility.

Reply:
Thanks, also fixed this in `release/cloudbuild_ci.yaml`.